clang/test/Driver/flang/flang.f90

```diff
@@ -19,32 +19,32 @@
 
 ! Check that f90 files are not treated as "previously preprocessed"
 ! ... in --driver-mode=flang.
-! RUN: %clang --driver-mode=flang -### -E                  %s 2>&1 | FileCheck --check-prefixes=ALL,CHECK-E %s
+! RUN: %flang -### -E                  %s 2>&1 | FileCheck --check-prefixes=ALL,CHECK-E %s
```

Comment:
This test is `UNSUPPORTED: classic_flang`. Did you verify that this will work correctly with LLVM Flang?

Reply:
No, I don't have any workable LLVM Flang build at my disposal.